apps/console/src/extensions/configs/application.tsx

```diff
@@ -34,6 +34,7 @@ export const applicationConfig: ApplicationConfig = {
     },
     attributeSettings: {
         advancedAttributeSettings: {
+            showHowToUseScopesDocLink: false,
```

Comment:
We don't need this anymore, right?

Reply:
resolved with https://github.com/wso2/identity-apps/pull/3640/commits/36d8f442d1333ae449424157d70d879c508e5d43